contracts/bridge/BridgeTransferCommon.sol

```diff
@@ -127,11 +127,17 @@ contract BridgeTransfer is BridgeHandledRequests, BridgeFee, BridgeOperator {
         if (_requestedNonce > upperHandleNonce) {
             upperHandleNonce = _requestedNonce;
         }
-        for (i = lowerHandleNonce; i <= upperHandleNonce && handleNoncesToBlockNums[i] > 0; i++) { }
-        lowerHandleNonce = i;
-        if (i != 0) {
-            recoveryBlockNumber = handleNoncesToBlockNums[i-1];
+        for (i = lowerHandleNonce; i <= upperHandleNonce && handleNoncesToBlockNums[i] > 0; i++) {
+            recoveryBlockNumber = handleNoncesToBlockNums[i];
+            delete handleNoncesToBlockNums[i];
+            delete closedValueTransferVotes[i];
```

Comment:
If we delete closedValueTransferVotes[i] here, how can we check and prevent duplicated voting?

Reply:
`lowerHandleNonceCheck(_requestedNonce)` can prevent it.